packages/meditrak-app/app/user/CreateUserPage.js

```diff
@@ -202,7 +202,7 @@ const fields = [
         I agree to the{' '}
         <Text
           style={localStyles.termsLink}
-          onPress={() => Linking.openURL('http://beyondessential.com.au/terms-and-privacy/')}
+          onPress={() => Linking.openURL('http://beyondessential.com.au/terms-and-conditions/')}
```

Comment:
```suggestion
          onPress={() => Linking.openURL('https://beyondessential.com.au/terms-and-conditions/')}
```
Tiny and probably doesn't matter since I think it'll always get switched to https, but might be nice?

Reply:
Thanks for the suggestion @rohan-bes! I wasn't sure which to go for and ended up just copying the format of the previous link but happy to implement your suggestion